acceptance/chaos_test.go

```diff
@@ -39,33 +38,31 @@ import (
 // independent keys, while nodes are being killed and restarted continuously.
 // The test measures not write performance, but cluster recovery.
 func TestChaos(t *testing.T) {
-	t.Skip("TODO(tschottdorf): currently unstable")
```

Comment:
oh, small side effect here. @BramGruneir should I take this out? I honestly thought we had re-enabled the Chaos test a while ago.